pootle/static/js/editor/mt/providers/CaighdeanTranslate.js

```diff
@@ -16,7 +16,7 @@ class CaighdeanTranslate extends MTProvider {
       apiKey,
       name: 'caighdean-translate',
       displayName: 'Caighdeán Translate',
-      url: 'http://borel.slu.edu/cgi-bin/seirbhis3.cgi',
+      url: '//borel.slu.edu/cgi-bin/seirbhis3.cgi',
       supportedLanguages: ['gd', 'gv', 'ga_IE', 'ga-IE'],
```

Comment:
im not sure if it would need to be recognised by the MT server - @kscanne does the server support plain `ga` ?